doc/documents/MLI_kernels/comm_lstm.rst

```diff
@@ -205,37 +192,45 @@ Where:
 Function Configuration Structure
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
```

Comment:
Please, move configuration structure description to Basic RNN section and left here only reference to it (at the moment basic rnn cites to the configuration structure, it should be swapped)